### DISCUSSION TITLE
Could not connect to the endpoint URL: "https://sts.amazonaws.com/"?

### DISCUSSION THREAD

**Discussion by fjlaile** [saleweaver/python-amazon-sp-api#411] [FAQ]
when i use "Oreder.get_orders(myorderid),it response :'EndpointConnectionError: Could not connect to the endpoint URL: "https://sts.amazonaws.com/"'  
could you help me ?

**Comment by timmyscj** [comment]
You misspelled Order

**Reply by eternalparquet** [reply]
If he mispelled Order in the source code it should have got the error: "name 'Oreder' is not defined".
I have the same problem.
